src/app/styles/theme.ts

```diff
@@ -0,0 +1,131 @@
+import { DefaultTheme } from 'styled-components';
+
+declare module 'styled-components' {
+  export interface DefaultTheme {
+    color: {
+      primaryBlue: string;
+      pointPink: string;
+      white: string;
+      black100: string;
+      black80: string;
+      gray50: string;
+      gray30: string;
+      blue50: string;
+      blue30: string;
+      blue10: string;
+    };
+    typo: {
+      title: {
+        fontFamily: string;
+        fontWeight: number;
+        fontSize: string;
+        lineHeight: string;
```

Comment:
```suggestion
import { DefaultTheme } from 'styled-components';
type ColorKeys = [
  'pointPink',
  'white',
  'black100',
  'black80',
  'gray50',
  'gray30',
  'blue50',
  'blue30',
  'blue10',
];

interface TypoProperties {
  fontFamily: string;
  fontWeight: number;
  fontSize: string;
  lineHeight: string;
}

interface DeviceSize {
  desktop: string;
  tablet: string;
  mobile: string;
}

declare module 'styled-components' {
  export interface DefaultTheme {
    color: {
      [key in ColorKeys[number]]: string;
    };
    typo: {
      title: {
         [key in keyof TypoProperties]: TypoProperties[key];
         }
      } //...
     deviceSize: {
        [key in keyof DeviceSize]: string;
    }; //...
    
```
type 과 interfacce 를 사용해서 `DefaultTheme` 을 설정 할 때,
타입 지정에 대해 좀 더 간편하게 할 수 있을 것 같습니다!

Reply:
[6e48aeb8a159ba45f21a68da0d04d984fbb33f58](url) 반영했습니다!

Reply:
https://github.com/GrebbMe/grabbme-fe/pull/18/commits/6e48aeb8a159ba45f21a68da0d04d984fbb33f58
반영했습니다!

Reply:
만들어진 theme 변수를 이용해서 타입을 잡아주면 되지 않을까요?